### PR TITLE
Rework initial explanation of shape tree structure and model

### PR DESCRIPTION
**Pull request by justinwb**
See the [Rendered version](https://deploy-preview-44--shapetrees-specification.netlify.app/)

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -178,56 +195,313 @@ data validation, and data migration.
...
-
-A { <code class="vocab">st:hasShapeTreeDecoratorIndex</code> <code>Si</code> }
-    [=arc=] indicates the location of an index of [SKOS hierarchies](#skos-graph) that describe
-    <code>ST</code>.
+* An { `ST` <code class="vocab">rdfs:label</code> `L` } [=arc=]
+    indicates that there is exactly one corresponding [=managed resource=] 
+    `R` in `STI` and it has the name `L`.
+* An { `ST` <code class="vocab">st:shape</code> `SH` }
+    [=arc=] indicates that [=managed resource=] `R` <em class="rfc2119">MUST</em>
+    have exactly one node which conforms to [=shape=] `SH`.
```
Comment (ericprud) [inline index.bs:485]: PROPOSE: s/MUST have/has/

File: index.bs [hunk 2]
```diff
@@ -257,6 +533,242 @@ A { <code class="vocab">st:hasShapeTreeDecoratorIndex</code> <code>Si</code> }
...
+
+A <dfn>shape tree locator</dfn> associates a [=managed resource=] with 
+one or more [=shape trees=]. No more than one [=shape tree locator=] may be
+associated with a [=managed resource=]. 
+
+For example, in [[Solid]] a [=shape tree locator=] would be stored in an
+[auxiliary resource](https://solidproject.org/TR/protocol#auxiliary-resources) 
+associated with a given [=managed resource=].
+
+A [=shape tree locator=] includes one or more [=shape tree locations=] via
```
Comment (ericprud) [inline index.bs:547]: I'm tempted to table-ify these definitions to make the list boundaries clear:

## Location properties

property | definition
-- | --
`st:node` | identifies the focus node for [=shape=] validation in the associated [=managed resource=]
`st:shape` | identifies the [=shape=] to which `st:node` must conform
`st:hasShapeTree` | the [=shape tree=] defining this [=shape tree instance=]
`st:hasRootShapeTree` | marks the primary, or parent [=shape tree=] in a physical hierarchy, and is set by the [Plant Operation](#plant-shapetree).
`st:hasShapeTreeInstanceRoot` | marks the primary, or parent [=shape tree instance=] in a physical hierarchy, and is set by the [Plant Operation](#plant-shapetree). <span class="note">@@ define as parm to PLANT?</span>

Note that A [=shape tree root=], and its corresponding [=shape tree instance root=] can be planted within an existing managed hierarchy, alongside or within other [=shape tree roots=] and [=shape tree instance roots=].
Properties like [=Shape tree locations=] in a given [=shape tree locators=] may have different focus nodes.

**Review by ericprud — CHANGES_REQUESTED** [review]

File: index.bs [hunk 3]
```diff
@@ -178,56 +195,313 @@ data validation, and data migration.
...
+      <td><code class="vocab">st:ShapeTreeContainer</code></td>
+      <td>RDF resource that contains other resources</td>
+    </tr>
+    <tr>
+      <td><code class="vocab">st:ShapeTreeNonRDFResource</code></td>
+      <td>Non-RDF resource such as binaries or images</td>
+    </tr>
+  </tbody>
+</table>
+
```
[the remark this answers is not in the document]
Comment (justinwb) [reply index.bs:246]: > a server-managed Resource that enumerates nested resources

A container resource isn't completely server-managed though, just the containment triples.

**Comment by ericprud** [comment]
for parallel-ism, propose s/st:hasShapeTreeInstanceRoot/st:hasRootShapeTreeInstance/

**Comment by justinwb** [comment]
> for parallel-ism, propose s/st:hasShapeTreeInstanceRoot/st:hasRootShapeTreeInstance/

Okay - made this change under the operations refactor branch to save another branch merge - https://github.com/shapetrees/specification/pull/45/commits/03e63e9a8a71fa2dbe0a64873328f4b0bd63799a

**Review by ericprud — APPROVED** [review]

**Pull request merged**